x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -179,3 +216,76 @@ NOTE: The PKI realm supports
 alternative to role mapping.
 
 --
+
+[float]
+[role="xpack"]
+[[pki-realm-for-proxied-clients]]
+==== PKI authentication for clients connecting to {kib}
+
+Clients must normally connect directly to {es} because the {es} PKI realm
+relies on the node's network interface to do the SSL/TLS authentication and to
+also internally forward the client's certificate chain for the realm to be able
+to optionally validate it. If SSL/TLS authenticatication is to be performed
+by {kib}, the PKI realm must be configured to permit delegation.
+
+Specifically, when clients presenting X.509 certificates connect to {kib},
+{kib} performs the SSL/TLS authentication. {kib} then forwards the client's
+certificate chain, by calling an {es} API, to have them further validated by
+the PKI realms that have been configured for delegation.
+
+To permit authentication delegation for a specific {es} PKI realm, start by
+configuring the realm for the usual case, as detailed in the
+<<pki-realm-for-direct-clients>>
+section. However, you must make sure that you have explicitly configured
+a `truststore` (or, equivalently `certificate_authorities`) even though it is
+the same trust configuration that you have configured on the network layer.
+Afterwards, simply toggle the `delegation.enabled` realm setting to `true`.
+This realm is now allowed to validate delegated PKI authentication (after
+restarting {es}).
+
+NOTE: PKI authentication delegation requires that
+the `xpack.security.authc.token.enabled` setting is `true`.
+
+NOTE: {kib} also needs to be configured to allow PKI certificate authentication.
+
+A PKI realm with `delegation.enabled` still works unchanged for clients
```

Comment:
Well... I'm not sure about it, since this very paragraph says that users (delegated or not) follow the same role mapping rules. However, I've made it a new paragraph for the mapping example that distinguishes delegated users. Maybe @lcawl can weigh in here too?